rbac/management/role/definitions/ocm_authenticated_user_local_test.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Do we need both of these files, or can we just have one `/role/definitions/ocm_local_test.json` file with both roles?

Reply:
Ah, right, let's put them in one file